install_dependencies.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+#######################################################################
+# STOP: If the dependency you want to add is required for the project # 
```

Comment:
This should probably go in the README as well

Reply:
Hmm, it's a pretty standard ROS feature. While I agree that people should know that,  there are lots of other things in ROS that are just as important, and I don't think we want to put them in, so I'm inclined to leave this out of the README for now.